lib/Gallery.pm

```diff
@@ -159,5 +160,24 @@ sub resize_image {
 	return $new_name;
 }
 
+sub exifdate {
+	my ($image_path) = @_;
+
+	return 'unused' unless 'date' eq $config->{sort_images_by};
```

Comment:
Trying to avoid adding unneeded dependencies and slowdowns to the default config. Maybe it's fine to compute unneeded dates for a bunch of images until proven otherwise?